From 8d2474c4774c6e7a220df05be6f772fc7e9ea2a0 Mon Sep 17 00:00:00 2001
From: Jason Zhu <jason.zhu@rock-chips.com>
Date: Wed, 5 Aug 2020 17:35:54 +0800
Subject: [PATCH] ARM: configs: Rename rv1126-tb-emmc.config to
 rv1126-tb.config

It is not need to separate storages with emmc, spi-nor and so on in
defconfig. Separate them in dts.

Update by:
make ARCH=arm rv1126_defconfig
cp .config rv1126.config
make ARCH=arm rv1126_defconfig rv1126-tb.config
make ARCH=arm menuconfig
scripts/diffconfig -m rv1126.config .config > arch/arm/configs/rv1126-tb.config

Usage:
make ARCH=arm rv1126_defconfig rv1126-tb.config

Signed-off-by: Jason Zhu <jason.zhu@rock-chips.com>
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I328091889c45ca1d09d2b2ce894b5a1d317d806d
---
 ...rv1126-tb-emmc.config => rv1126-tb.config} | 29 ++++++++++---------
 1 file changed, 15 insertions(+), 14 deletions(-)
 rename arch/arm/configs/{rv1126-tb-emmc.config => rv1126-tb.config} (98%)

diff --git a/arch/arm/configs/rv1126-tb-emmc.config b/arch/arm/configs/rv1126-tb.config
similarity index 98%
rename from arch/arm/configs/rv1126-tb-emmc.config
rename to arch/arm/configs/rv1126-tb.config
index a5fa165d11a1..d00c782a98de 100644
--- a/arch/arm/configs/rv1126-tb-emmc.config
+++ b/arch/arm/configs/rv1126-tb.config
@@ -1,6 +1,6 @@
 CONFIG_BLK_DEV_SD=m
-CONFIG_BT_HCIUART=m
 CONFIG_BT=m
+CONFIG_BT_HCIUART=m
 CONFIG_CFG80211=m
 CONFIG_COMMON_CLK_PROCFS=y
 # CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
@@ -11,7 +11,6 @@ CONFIG_DTC_OMIT_DISABLED=y
 CONFIG_DTC_OMIT_EMPTY=y
 # CONFIG_DTC_SYMBOLS is not set
 CONFIG_DWMAC_ROCKCHIP=m
-CONFIG_INITCALL_ASYNC=y
 # CONFIG_INPUT_TOUCHSCREEN is not set
 CONFIG_MAC80211=m
 CONFIG_MII=m
@@ -23,36 +22,32 @@ CONFIG_RFKILL=m
 CONFIG_RFKILL_RK=m
 CONFIG_ROCKCHIP_HW_DECOMPRESS=y
 CONFIG_ROCKCHIP_RAMDISK=y
-CONFIG_ROCKCHIP_THUNDER_BOOT_MMC=y
 CONFIG_ROCKCHIP_THUNDER_BOOT=y
-CONFIG_ROMFS_BACKED_BY_BLOCK=y
-# CONFIG_ROMFS_BACKED_BY_BOTH is not set
-# CONFIG_ROMFS_BACKED_BY_MTD is not set
 CONFIG_ROMFS_FS=y
-CONFIG_ROMFS_ON_BLOCK=y
 CONFIG_SCSI=m
 CONFIG_SCSI_MOD=m
 # CONFIG_SLUB_SYSFS is not set
+CONFIG_SND=m
 CONFIG_SND_ALOOP=m
 CONFIG_SND_DMAENGINE_PCM=m
 CONFIG_SND_HRTIMER=m
-CONFIG_SND=m
 CONFIG_SND_PCM=m
 CONFIG_SND_SIMPLE_CARD=m
 CONFIG_SND_SIMPLE_CARD_UTILS=m
+CONFIG_SND_SOC=m
 CONFIG_SND_SOC_DUMMY_CODEC=m
 CONFIG_SND_SOC_I2C_AND_SPI=m
-CONFIG_SND_SOC=m
 CONFIG_SND_SOC_RK817=m
+CONFIG_SND_SOC_ROCKCHIP=m
 CONFIG_SND_SOC_ROCKCHIP_I2S=m
 CONFIG_SND_SOC_ROCKCHIP_I2S_TDM=m
-CONFIG_SND_SOC_ROCKCHIP=m
 CONFIG_SND_SOC_ROCKCHIP_PDM=m
 CONFIG_SND_TIMER=m
 CONFIG_SOUND=m
 CONFIG_STMMAC_ETH=m
 CONFIG_STMMAC_PLATFORM=m
 # CONFIG_TEE is not set
+CONFIG_USB=m
 # CONFIG_USB_CONFIGFS_F_UAC1 is not set
 # CONFIG_USB_CONFIGFS_F_UAC2 is not set
 # CONFIG_USB_CONFIGFS_F_UVC is not set
@@ -63,9 +58,6 @@ CONFIG_USB_DWC3_ROCKCHIP_INNO=m
 CONFIG_USB_EHCI_HCD=m
 CONFIG_USB_EHCI_HCD_PLATFORM=m
 CONFIG_USB_HID=m
-# CONFIG_USB_KBD is not set
-CONFIG_USB=m
-# CONFIG_USB_MOUSE is not set
 # CONFIG_USB_NET_DRIVERS is not set
 CONFIG_USB_OHCI_HCD=m
 CONFIG_USB_OHCI_HCD_PLATFORM=m
@@ -75,5 +67,14 @@ CONFIG_USB_XHCI_PLATFORM=m
 # CONFIG_VIDEO_GC4C33 is not set
 # CONFIG_VIDEO_IMX347 is not set
 # CONFIG_VIDEO_OV4689 is not set
-CONFIG_VIDEO_ROCKCHIP_THUNDER_BOOT_ISP=y
 CONFIG_WL_ROCKCHIP=m
+CONFIG_INITCALL_ASYNC=y
+CONFIG_ROCKCHIP_THUNDER_BOOT_MMC=y
+CONFIG_ROCKCHIP_THUNDER_BOOT_SFC=y
+CONFIG_ROMFS_BACKED_BY_BLOCK=y
+# CONFIG_ROMFS_BACKED_BY_BOTH is not set
+# CONFIG_ROMFS_BACKED_BY_MTD is not set
+CONFIG_ROMFS_ON_BLOCK=y
+# CONFIG_USB_KBD is not set
+# CONFIG_USB_MOUSE is not set
+CONFIG_VIDEO_ROCKCHIP_THUNDER_BOOT_ISP=y
-- 
2.35.3

